File: src/codility_temp/lesson5/CountDiv.java
package codility_temp.lesson5;

public class CountDiv {

    public int solution(int A, int B, int K) {

        if (B == 0) {
            return 1;
        }

        int ans = B / K;

        if (A == 0) {
            return ans + 1;
        }

        ans -= A / K;
        if (A % K == 0) {
            ans++;
        }

        return ans;

    }

    public int solution1(int A, int B, int K) {
        if (A == 0) {
            return (B / K) + 1;
        }
        else {
            return (B/K) - ((A-1)/K);
        }

    }
}

File: src/codility/kotlin/CountDiv.kt
package codility.kotlin

fun main() {
    print(0 % 7)
}

class CountDiv {



    fun solution(A: Int, B: Int, K: Int): Int {
        return if (A == 0) B/ K + 1 else B/K - (A-1) / K
    }

}